_topic_maps/_topic_map.yml

```diff
@@ -1317,6 +1317,8 @@ Topics:
     File: installing-operators
   - Name: Observing the network traffic
     File: observing-network-traffic
+  - Name: Understanding Network Observability Operator
```

Comment:
@xenolinux do you think we should move this in the topic map under the Overview instead of under Observing?